release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -1174,6 +1179,14 @@ With this release, replacement control plane nodes are assigned to the correct i
 
 * The previous resolution of link:https://issues.redhat.com/browse/OCPBUGS-5546[OCPBUGS-5546] removed the `clusterName` assignment of `MachineConfig.Name` in the machine configuration object. As a result, the value of the parameter was an empty string and, when it was combined with the value of `machineName` to create an IP address name, it created an invalid value. The invalid value caused machines to fail during provisioning. With this release, the value for `clusterName` is obtained from the infrastructure object so that it creates a valid IP address name. (link:https://issues.redhat.com/browse/OCPBUGS-7696[*OCPBUGS-7696*])
 
+* The Kubernetes 1.26 release introduced changes to the node infrastructure, such as removing an unhealthy node with an `NotReady` status from the public load balancer to prevent the node from receiving routing traffic. These changes can impact a {product-title} node that runs inside a cluster on Microsoft Azure, because removing a node from a public load balancer can result in the node losing internet connectivity. This issue might render the node unable to regain a ready status and establish an outbound connection. The {product-title} {product-version} release fixes this issue  so that a node marked with `NotReady` status is detected by the `kube-proxy` based health probes, now the default for services of `type=LoadBalancer` with `externalTrafficPolicy=Cluster`, without the need of node detachment from the public load balancer. This means that a node can retain an outbound internet connection throughout these phases. (link:https://issues.redhat.com/browse/OCPBUGS-7359[*OCPBUGS-7359*])
+
+[discrete]
+[id="ocp-4-13-cloud-cred-operator-bug-fixes"]
+==== Cloud Credential Operator
+
+* Amazon S3 changed their Amazon S3 bucket configuration, so that a bucket created in a Amazon Web Services (AWS) region has S3 Block Public Access enabled and access control limits (ACLs) disabled by default. This configuration limits S3 bucket resources to private use. The {product-title} (product-version} updates the CCO utility (`ccoctl`) to account for the default S3 bucket configuration so that S3 bucket resources are publicly available. (link:https://issues.redhat.com/browse/OCPBUGS-11706[*OCPBUGS-11706*])
```

Comment:
```suggestion
* Previously, Amazon Simple Storage Service (Amazon S3) changed their Amazon S3 bucket configuration, so a bucket created in a Amazon Web Services (AWS) region has S3 Block Public Access enabled and access control limits (ACLs) disabled by default. This configuration limits S3 bucket resources to private use. {product-title} (product-version} updates the CCO utility (`ccoctl`) to account for the default S3 bucket configuration so that S3 bucket resources are publicly available. (link:https://issues.redhat.com/browse/OCPBUGS-11706[*OCPBUGS-11706*])
```

Reply:
Okay it might be better to do 

> Amazon Simple Storage Service (Amazon S3) updated the Amazon S3 bucket configuration so a bucket created in a Amazon Web Services (AWS) region has S3 Block Public Access enabled and access control limits (ACLs) disabled by default.

Just a thought

Reply:
Updated.